rtbkit/core/banker/go_account.cc

```diff
@@ -208,6 +208,15 @@ GoAccounts::updateBalance(const AccountKey &key, int64_t newBalance)
     account->router->balance = newBalance;
 }
 
+int64_t
```

Comment:
Except that we're not in Redis here so we should be using the Amount class when dealing with monetary values. Otherwise, I have to care about whether what the scale of the value is which can only lead to tears.